docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -55,7 +55,7 @@ The server element enables you to easily represent a single server, a stack of s
 
 ## Connections
 
-When building a canvas panel, you can connect elements together to create more complex visualizations. Connections are created by dragging from the connection anchor of one element to the connection anchor of another element. You can also create connections to the background of the canvas panel. Connection anchors are displayed when you hover over an element and inline editing is turned on. To remove a connection, simply click on the connection directly and then press the "Delete" or "Backspace" key.
+When building a canvas panel, you can connect elements together to create more complex visualizations. Connections are created by dragging from the connection anchor of one element to the connection anchor of another element. You can also create connections to the background of the canvas panel. Connection anchors are displayed when you hover over an element and [inline editing](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/canvas/#inline-editor) is turned on. To remove a connection, simply click on the connection directly and then press the "Delete" or "Backspace" key.
```

Comment:
I think the section you're trying to link to is the "Inline editing" section rather than the "Inline editor" section. The "Inline editing" section has the gif that shows the feature being toggled off.

```suggestion
When building a canvas panel, you can connect elements together to create more complex visualizations. Connections are created by dragging from the connection anchor of one element to the connection anchor of another element. You can also create connections to the background of the canvas panel. Connection anchors are displayed when you hover over an element and [inline editing](#inline-editing) is turned on. To remove a connection, simply click on the connection directly and then press the "Delete" or "Backspace" key.
```